yoga/Yoga.c

```diff
@@ -229,8 +229,6 @@ static YGLogger gLogger = &YGDefaultLog;
 static inline const YGValue *YGComputedEdgeValue(const YGValue edges[YGEdgeCount],
                                                  const YGEdge edge,
                                                  const YGValue *const defaultValue) {
-  YG_ASSERT(edge <= YGEdgeEnd, "Cannot get computed value of multi-edge shorthands");
```

Comment:
@emilsjolander So If I'm understanding you right we should:

1. Add the `ASSERT` back.
2. Change all the tests where we access `YGEdgeAll`, `YGEdgeHorizontal`, `YGEdgeVertical` to access their respective representations.
3. Remove the new `YGEdgeTests` since they are testing functionality we don't support.